src/Form/Control/DropdownCascade.php

```diff
@@ -43,7 +43,7 @@ public function init(): void
 
         $this->cascadeInput = is_string($this->cascadeFrom) ? $this->form->getControl($this->cascadeFrom) : $this->cascadeFrom;
 
-        if (!$this->cascadeInput instanceof Field) {
+        if (!$this->cascadeInput->field instanceof Field) {
             throw new Exception('cascadeFrom property should be an instance of ' . Field::class);
```

Comment:
I believe the lines should be changed to `instance of Form\Control::class`

There are more changes to do in this class property names to keep consistency so I will take it up later today.

Reply:
```suggestion
            if (!$this->cascadeInput instanceof Control) {
```